Userland/Libraries/LibGUI/Window.cpp

```diff
@@ -321,14 +321,6 @@ void Window::set_window_type(WindowType window_type)
     }
 }
 
-void Window::make_window_manager(unsigned event_mask)
-{
-    (void)event_mask;
-    // FIXME: delete this function
-    // GUI::ConnectionToWindowManagerServer::the().async_set_event_mask(event_mask);
-    // GUI::ConnectionToWindowManagerServer::the().async_set_manager_window(m_window_id);
-}
-
```

Comment:
Removing the function here seems arbitrary. It doesn't really fit in the scope of the commit and is not clear from its message. Also trying to compile serenity after this would fail as Keymap still relies on the function

Reply:
my bad